gammapy/maps/wcsnd.py

```diff
@@ -460,51 +460,150 @@ def downsample(self, factor, preserve_counts=True):
             data /= factor**2
         return self.__class__(geom, data, meta=copy.deepcopy(self.meta))
 
-    def plot(self, ax=None, idx=None, **kwargs):
-        """Quickplot method.
+
+
+    
+    def plot(self, ax=None, idx=None, fig=None, add_cbar=False, stretch='linear', smooth=None, radius=1, **kwargs):
+        """
+        Plot image on matplotlib WCS axes
 
         Parameters
         ----------
-        norm : str
-            Set the normalization scheme of the color map.
+        ax : `~astropy.visualization.wcsaxes.WCSAxes`, optional
+            WCS axis object to plot on.
         idx : int or tuple
             Set the image slice to plot if this map has non-spatial dimensions.
             For maps with exactly one non-spatial dimension idx can be an int
-        **kwargs : dict
-            Keyword arguments passed to `~matplotlib.pyplot.imshow`.   
+        TODO: let idx work with slicing
+
+        fig : `~matplotlib.figure.Figure`, optional
+            Figure
+        stretch : str, optional
+            Scaling for image ('linear', 'sqrt', 'log').
+            Similar to normalize and stretch functions in ds9.
+            See http://docs.astropy.org/en/stable/visualization/normalization.html
+
+        smooth: ~string : either 'gauss' or 'disk' 
```

Comment:
Please change the documentation of the smoothing parameters to the same format  as the other parameters of the `.plot()` function.

Reply:
Suggest to use this format for multiple-choice option parameters:
```
When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:

order : {'C', 'F', 'A'}
    Description of `order`.
```
https://numpydoc.readthedocs.io/en/latest/format.html